main/UseCases/InterviewMaker.java

```diff
@@ -120,13 +94,63 @@ public void hireIntern(){
      * @return a String representation of the interviewIntern info.
      */
     public String getInterviewInternInfo(){
-        return currentHRSystem.getInterviewInternNames();
+        StringBuilder res = new StringBuilder();
+
+        ResponseTree<ArrayList<String>> responseTree = this.currentInterviewIntern.getResponseTree();
+        for (ResponseTree<ArrayList<String>> response : responseTree.getChildren()){
+            // if the response tree is a root, display the interns initial response(i.e. name, age & skill info).
+            if (response.isRoot()){
+                res.append(response.getData());
+            }
+            res.append(" ");
```

Comment:
But here if the response isRoot only the first if statement will evaluate to true. The second can't be true at the same time, so you are only returning their info, and no player options.

Reply:
Does it mean that when we call getChildren() on the interns response Tree, the first and only thing it obtains is the root?. From the type, it returns something in the form ResponseTree<ArrayList<String>>, is this referring to only the root?

Reply:
sorry, it returns ResponseTree<ArrayList<String>> **

Reply:
"ResponseTree<ArrayList<String>>" *

Reply:
Sorry it returns, ResponseTree<ArrayList< "string" >>

Reply:
Sorry, I don't really know what you mean, since I haven't looked into ResponseTree. I just need you to confirm: the very first thing that we return to the player, it has the interns info. first sentence, and also the player's choice, no? If not, talk to camille and see how you could change this. I can't say much because I don't know what's in response tree. I only found this because the two if statements are opposite of each other, so it is not possible to be evaluated to true at the same time

Reply:
Maybe just get rid of the second if statement? Since no matter when, we always want to return the player's options...? I dunno, up to you. Also any progress with finding the null using Debugger?

Reply:
Yes it has the player's choice ideally. The two conditions are bc we are looping through all the branches in the tree so a branch might be a root or it might not be a root (it is not meant to eval to both true, if a branch is the root then we add that root info i.e. the name etc. to the response and is a branch is not a root i.e. a response tree then add that choice to the result and then output the result of both the name + choice info.) I'll take to Camille about the structure bc it looks like when we loop, the branch is empty so it's not returning any information.

Reply:
oh, so nothing is generated properly then? ok, yea try to find camile to solve this
Then in SPhase right now, you should be able to test other commands in InterviewLevel (If you have any)

Reply:
yeah, that's the problem, so then the startOfInterviewPrompt isn't generating well. Alright, I'll try the other prompts!

Reply:
sPhase is stuck at that point, no matter the input I give

Reply:
ah okie,, look into GameGenerator to see if the interns are actually assigned a non empty ResponseTree then,, and try to get Camille and Farzana to solve.

Reply:
ahh yeah i think I see the problem, rn responsetree has to be set manually by input, ill see if we can resolve this!

Reply:
ok! I'll merge this pull request for now then ;)